src/protocols/tacacs/encode.c

```diff
@@ -182,7 +187,7 @@ ssize_t fr_tacacs_encode(uint8_t *buffer, size_t buffer_len, uint8_t const *orig
 
 	if (!vp) {
 		if (!original) {
-			fr_strerror_printf("No TACACS+ %s in the attribute list",
+			fr_strerror_printf("decode: No TACACS+ %s in the attribute list",
```

Comment:
But we're encoding... Maybe just use __FUNCTION__?

Reply:
Good idea.